pkgs/dart_model/lib/src/macro_metadata.g.dart

```diff
@@ -1513,24 +1497,24 @@ extension type Instantiation.fromJson(Map<String, Object?> node)
 ///
 extension type IntegerLiteral.fromJson(Map<String, Object?> node)
     implements Object {
-  static final TypedMapSchema _schema = TypedMapSchema({
-    'text': Type.stringPointer,
-  });
-  IntegerLiteral({String? text})
-    : this.fromJson(Scope.createMap(_schema, text));
+  IntegerLiteral({String? text, int? value})
```

Comment:
not sure why [`IntegerLiteral`](https://github.com/dart-lang/sdk/blame/f34a2ed99fc1b34cedbd974a5801f8d922121126/pkg/_fe_analyzer_shared/lib/src/metadata/expressions.dart#L93) has nullable fields but [`DoubleLiteral`](https://github.com/dart-lang/sdk/blame/f34a2ed99fc1b34cedbd974a5801f8d922121126/pkg/_fe_analyzer_shared/lib/src/metadata/expressions.dart#L108) has non-nullable? this causes some tests to fail and I'm not sure what to make of it